charts/traction/values-e79518-dev.yaml

```diff
@@ -27,18 +35,28 @@ acapy:
       memory: 400Mi
   tails:
     enabled: true
+  staticArgs:
+    walletName: mywallet
+
 postgresql:
   fullnameOverride: 'traction-dts'
-  resources:
-    limits:
-      cpu: 200m
-      memory: 820Mi
-    requests:
-      cpu: 120m
-      memory: 400Mi
+  auth:
+    postgresqlUsername: postgres
```

Comment:
Is this using `postgres` as admin user? Shouldn't we use an unprivileged user instead? This also seems undocumented in the bitnami chart repo (I see `auth.postgresqlPassword`, but not username - it might be due to a newer version? See https://github.com/bitnami/charts/tree/main/bitnami/postgresql#global-parameters